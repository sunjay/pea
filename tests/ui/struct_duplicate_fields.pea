struct OhNo {
    ahhh: [u8],
    ahhh2: i64,
    ahhh: u8,
    ahhh: [i64],
}

struct Person {
    age: i64,
}

fn main() {
    let person = Person {
        age: 32,
        age: 32,
    };
}
