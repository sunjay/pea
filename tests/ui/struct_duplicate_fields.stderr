[tests/ui/struct_duplicate_fields.pea:4:5-4:8] error: field `ahhh` is already declared
[tests/ui/struct_duplicate_fields.pea:2:5-2:8] info: previous declaration of field `ahhh`
[tests/ui/struct_duplicate_fields.pea:4:5-4:8] error: `ahhh` redefined here

[tests/ui/struct_duplicate_fields.pea:5:5-5:8] error: field `ahhh` is already declared
[tests/ui/struct_duplicate_fields.pea:4:5-4:8] info: previous declaration of field `ahhh`
[tests/ui/struct_duplicate_fields.pea:5:5-5:8] error: `ahhh` redefined here

[tests/ui/struct_duplicate_fields.pea:15:9-15:15] error: field `age` specified more than once

error: aborting due to 3 previous errors

